Datadog/Datadog.xcodeproj/project.pbxproj

```diff
@@ -1479,7 +1479,7 @@
 		61363D9C24D999F70084CD6F /* DDError.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = DDError.swift; sourceTree = "<group>"; };
 		61363D9E24D99BAA0084CD6F /* DDErrorTests.swift */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.swift; path = DDErrorTests.swift; sourceTree = "<group>"; };
 		61378BA72555329E00F28837 /* DatadogTests.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = DatadogTests.xcconfig; sourceTree = "<group>"; };
-		61378BB22555337900F28837 /* DatadogSDKTesting.local.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; path = DatadogSDKTesting.local.xcconfig; sourceTree = "<group>"; };
+		61378BB22555337900F28837 /* DatadogSDKTesting.local.xcconfig */ = {isa = PBXFileReference; lastKnownFileType = text.xcconfig; name = DatadogSDKTesting.local.xcconfig; path = ../xcconfigs/DatadogSDKTesting.local.xcconfig; sourceTree = SOURCE_ROOT; };
```

Comment:
There seems to be a difference in how different versions of Xcode resolve their `xcconfig` paths. [Following this advice](https://developer.apple.com/forums/thread/690443), I'm changing `*.local.xcconfig` paths to "Relative to Project" (vs "Relative to Group" before). This is to solve another CI issue after Xcode stack upgrade (to 14.x):

```
Testing failed:
	could not find included file '../xcconfigs/MockServerAddress.local.xcconfig' in search paths
	Testing cancelled because the build failed.
```

Note - this only relates to `*.local.xcconfig` files, which are referenced using weak import from regular `.xcconfig` files. Regular ones are configured in `pbxproj` directly, so Xcode doesn't have problems with their visibility.